qbpm/main.py

```diff
@@ -21,6 +20,7 @@ def creator_options(f: Callable[..., Any]) -> Callable[..., Any]:
                 is_flag=True,
                 help="If --launch is set, run qutebrowser in the foreground.",
             ),
+            # TODO --no-icon?
```

Comment:
Not sure if I want to make a network connection by default without asking. Maybe `--icon` should be explicitly required.